pkg/conn/basedb.go

```diff
@@ -64,10 +64,19 @@ func (d *DefaultDBProviderImpl) Apply(config config.DBConfig) (*BaseDB, error) {
 
 	doFuncInClose := func() {}
 	if config.Security != nil && len(config.Security.SSLCA) != 0 {
-		tlsConfig, err := toolutils.ToTLSConfig(config.Security.SSLCA, config.Security.SSLCert, config.Security.SSLKey)
+		if err := config.Security.LoadTLSContent(); err != nil {
```

Comment:
there're many places calling this method, which one do you expect that `LoadTLSContent` loads the certificates?

Reply:
I want every call to use `LoadTLSContent` because  in L71, i will always use cert's bytes to init tls conn

```go
tlsConfig, err := toolutils.ToTLSConfigWithVerifyByRawbytes(config.Security.SSLCABytes,
	config.Security.SSLCertBytes, config.Security.SSLKEYBytes, []string{})
`` `